explore-analyze/elastic-inference/eis.md

```diff
@@ -5,6 +5,82 @@ applies_to:
 navigation_title: Elastic Inference Service (EIS)
 ---
 
-# Elastic {{infer-cap}} Service
+# Elastic {{infer-cap}} Service [elastic-inference-service-eis]
 
-This is the documentation of the Elastic Inference Service.
+The Elastic {{infer-cap}} Service (EIS) enables you to leverage AI-powered search as a service without deploying a model in your cluster.
+With EIS, you don't need to manage the infrastructure and resources required for large language models (LLMs) by adding, configuring, and scaling {{ml}} nodes.
+Instead, you can use {{ml}} models in high-throughput, low-latency scenarios independently of your {{es}} infrastructure.
+
+% TO DO: Link to the EIS inference endpoint reference docs when it's added to the OpenAPI spec. (Comming soon) %
+
+## Available task types
+
+EIS offers the following {{infer}} task types to perform:
+
+* Chat completion
```

Comment:
```suggestion
* `chat_completion`
```
Just because the [Inference API docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/put-inference-api.html#put-inference-api-desc) show all task types as code.